src/config.js

```diff
@@ -0,0 +1,39 @@
+export const config = {
+  state: {
+    name: "",
+    initialStatus: "fine",
+    initialFoodPercent: "100",
```

Comment:
why is it a string? could be a number, because it's percent

Reply:
fixed